frontend/tests/helpers/mapbox/mapbox-stub-helpers.js

```diff
@@ -0,0 +1,28 @@
+// @registeredEvents is an object hash associating one or more actions handlers to a
```

Comment:
LOVE THE COMMENTS! I think a more standard way of documenting functions is:
```
/**
 * Block comment describing the function blah blah blah
 * @param [optional param type] ParamName [optional param description]
 */
```

but also love that you made this 🏕 !

Reply:
check out https://devdocs.io/jsdoc/

Reply:
Coooool thx, It does make me feel warm and fuzzy following style guides 😄